### DISCUSSION TITLE
UNO Platform - Build a Todo App - Part 4

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#51] [General]
# UNO Platform - Build a Todo App - Part 4

In the forth part we will take care of our newly created todo item. Until now we clicked on a button and that's all. Sure we have a nice form where we can enter the basic information, but that is all.
The time has come to put our element into the respective swimlane. We will also learn how to add further nuget packages to our project.

https://steven-giesel.com/blogPost/2d96d970-ef11-48f4-a102-9339fc362a75

**Comment by darkoverlordofdata** [comment]
I’ve been trying to learn how to code Uno on Linux. Not only does this demo actually work, you explain what you're doing, and why! Since I don’t have access to VS on Linux, this is essential to learning the framework. 
Thank you - I was starting to think Uno was all smoke and mirrors.

**Reply by linkdotnet** [reply]
Hey @darkoverlordofdata thank you so much for those kind words!

I would give it a shot under Linux, as there are not that many alternatives.
